mbed/mbed.py

```diff
@@ -177,6 +177,22 @@ def progress():
     sys.stdout.flush()
     sys.stdout.write('\b')
 
+def show_progress(title, percent, max_width=80):
+    percent = round(float(percent), 2)
+    show_percent = '%.2f' % percent
+    line = str(title) + ' |'
+    bwidth = max_width - len(str(title)) - len(show_percent) - 6
+    for i in range(0, bwidth):
+        line += '#' if i <= (percent / 100 * bwidth) else '-'
+    line += '| ' + show_percent + '%'
+    sys.stdout.write(line+'\r')
+    sys.stdout.flush()
+
+def hide_progress(max_width=80):
+    line = ''
+    for i in range(0, max_width):
+        line += ' '
```

Comment:
These 3 lines could be:
```python
line = ' ' * max_width
```
Up to you. I have a preference for the above because I think it better conveys the intent.

Reply:
Great that you spotted this. Will change it shortly.